angular-client/src/components/history/carousel.component.html

```diff
@@ -0,0 +1,21 @@
+<p-carousel [value]="runs" [numVisible]="1" [numScroll]="1" [circular]="true" [showIndicators]="false">
+  <ng-template let-run pTemplate="item">
+    <div class="runcard">
+      <div class="row">
+        <h3 class="left">Run #{{ run.id }}</h3>
+        <h3 class="right">Driver: {{ run.driverName }}</h3>
+      </div>
+      <div class="row">
+        <h3 class="left">{{ run.time }}</h3>
+        <h3 class="right">System: {{ run.systemName }}</h3>
+      </div>
+      <div class="row">
+        <h3 class="right">Location: {{ run.locationName }}</h3>
+      </div>
+    </div>
+  </ng-template>
+</p-carousel>
+<div class="vertical-center">
+  <button mat-button (click)="onNoClick()">Back</button>
```

Comment:
can you not just call mat-button?

Reply:
the tag <button mat-button> is the correct usage according to the docs

Reply:
<img width="665" alt="Screen Shot 2023-11-06 at 8 09 20 PM" src="https://github.com/Northeastern-Electric-Racing/Argos/assets/102067727/d6f54853-5cbd-475b-8744-adbfda37465c">
examples